components/TaskCard/index.js

```diff
@@ -1,25 +1,31 @@
 import { useState } from "react";
 
-import { Tasks, Task, TaskListItem } from "./TaskCard.styled";
+import { Tasks, Task, TaskListItem, Label, Labels } from "./TaskCard.styled";
 
 export const TaskCard = ({ project, index }) => {
   const [visible, setVisible] = useState(false);
   const projectClickHandler = () => {
     setVisible(!visible);
   };
+  console.log(project, index);
+  const { projectTitle, tasks, projectDue, projectSize } = project;
   return (
     <TaskListItem onClick={projectClickHandler}>
       <b>
         {`P${index + 1}: `}
-        {project.projectTitle}
+        {projectTitle}
       </b>
       {visible && (
         <Tasks>
-          {project.tasks.map((task, index) => (
+          {tasks.map((task, index) => (
             <Task key={index}>{task}</Task>
           ))}
         </Tasks>
       )}
+      <Labels>

```

Comment:
the name "Labels" let me think that we are in a form right now. Maybe you could change the name to "TopicHeading" or "SingleHeadline".

Reply:
It's just the name as I see it. in general I am glad that there are no errors. Thanks, it's always nice to hear opinions.

Reply:
haha i feel you !